dashboard/test/specs/keys.js

```diff
@@ -0,0 +1,49 @@
+const chain = require('chain-sdk')
+const uuid = require('uuid')
+
+const client = new chain.Client()
+let signer
+
+describe('mock hsm keys', () => {
+  describe('list view', () => {
+    before(() => {
+      signer = new chain.HsmSigner()
+
+      return expect(ensureConfigured()).to.be.fulfilled
+        .then(() => expect(setUpObjects(signer)).to.be.fulfilled)
+        .then(() => browser.url('/mockhsms'))
+    })
+
+    it('does not display a welcome message', () => {
+      browser.isExisting('.component-EmptyList').should.equal(false)
```

Comment:
*ducking for cover*

Eyeballing it right now, the `component-` prefix sure is verbose, isn't it? I'm down to get rid of it if you are. UpperCamelCase seems obvious enough as an indicator of a component, and there shouldn't be collisions unless we adopt a CSS framework that uses that casing style.